src/modules/dashboard/components/DomainDropdown/DomainSelectItem.tsx

```diff
@@ -78,10 +78,12 @@ const DomainSelectItem = ({
              */}
             <ColorTag color={color || Color.LightPink} />
           </div>
-          <Heading
-            appearance={{ margin: 'none', size: 'normal', theme: 'dark' }}
-            text={name}
-          />
+          <div className={styles.headingWrapper}>
+            <Heading
+              appearance={{ margin: 'none', size: 'normal', theme: 'dark' }}
+              text={name}
+            />
+          </div>
           {ethDomainId === ROOT_DOMAIN_ID && (
```

Comment:
This is the only thing I could find, the domain names in the dropdown, now are all "smooshed" together to the left,
and there's no space between them and the color:

![Screenshot from 2021-02-03 15-12-09](https://user-images.githubusercontent.com/1193222/106752221-def61800-6632-11eb-9858-f42c733d2ceb.png)